### PR TITLE
Migrate command configs to `composite_forest`

### PR DESCRIPTION
**Pull request by uvsmtid-gs**
*   Migrate configs to `composite_forest`:
    *   Remove `func_selector_tree` from schema.
    *   Remove `jump_tree` from schema.
    *   Remove `interp_selector_tree` from schema.
    *   Remove `first_arg_vals_to_next_interp_factory_ids` from schema.
    *   Remove `tree_abs_path_to_interp_id` from schema.
*   Add `lay` command as the new default.
*   Make `@/exe/check_env.bash` Bash phase available as a copy to be runnable even before `@/exe/bootstrap_env.bash`.
*   Install `@/exe/argrelay_common_lib.bash` as symlink unconditionally.
*   Improve `check_env`:
    *   Bash-phase to check `@/exe/argrelay_common_lib.bash`.
    *   Implement Bash-phase `check_env_plugin.git_version.bash`. 
    *   Implement Python-phase `PluginCheckEnvArgrelayUnversionedSymlinks.py`.
    *   Display time since `@/exe/dev_shell.bash` start.
*   Rename to `@/exe/script_plugin.d/check_env_plugin.all_argrelay_plugins.bash`.
*   FS_92_75_93_01: explain inability to query `argrelay`-linked commands within complex bash command lines.

### CODE REVIEW

File: docs/feature_stories/FS_36_17_84_44.check_env.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ feature_title: check_env script
 feature_status: TEST
 ---
 
+TODO: Explain that `check_env` script is copied into target project just like `boostrap_env` script.
```
Comment (uvsmtid) [inline docs/feature_stories/FS_36_17_84_44.check_env.md:7]: Do before merge.

File: src/argrelay/_version.py [hunk 2]
```diff
@@ -2,4 +2,4 @@
 # See `docs/dev_notes/version_format.md`:
 # Implements this:
 # https://stackoverflow.com/a/7071358/441652
-__version__ = "0.7.11"
+__version__ = "0.7.12.dev2"
```
Comment (uvsmtid) [inline src/argrelay/_version.py:5]: Finalize before merge.

File: docs/feature_stories/FS_36_17_84_44.check_env.md (hunk 1 above)
Comment (uvsmtid-gs) [reply docs/feature_stories/FS_36_17_84_44.check_env.md]: Done

File: src/argrelay/_version.py (hunk 2 above)
Comment (uvsmtid-gs) [reply src/argrelay/_version.py]: Done

**Review by uvsmtid — APPROVED** [review]

**Pull request merged**